frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -53,10 +53,6 @@ import { createSegments, mapSnapshotsToWindowId } from './utils/segmenter'
 const IS_TEST_MODE = process.env.NODE_ENV === 'test'
 const BUFFER_MS = 60000 // +- before and after start and end of a recording to query for.
 const DEFAULT_REALTIME_POLLING_MILLIS = 3000
-const REALTIME_POLLING_PARAMS = {
-    source: SnapshotSourceType.realtime,
-    version: '2',
-}
 
```

Comment:
are they the default if not provided?

Reply:
Good point, I'm not super familiar. I ported it directly from the other PR but that drops the `source`. `version` will be correct in both cases.

Going to add the `source` back in case it is needed: https://github.com/PostHog/posthog/pull/21207/files#r1542936549